### PR TITLE
Adds FAQ section

### PR DESCRIPTION
**Pull request by kinduff**
Includes a fix to the error related to GraphQL not able to find subfields from the image key

Documenting the issue from Gatsby https://github.com/gatsbyjs/gatsby/issues/5990

### CODE REVIEW

**Comment by danielmahon** [comment]
Thanks @kinduff , I tried to make this apparent in the comments of the code example, but I suppose it wouldn't hurt to clarify it further. Also, you no longer need the `options.name` property. I think it's been removed completely in the latest release.

**Comment by kinduff** [comment]
@danielmahon sweet! Updated.

File: README.md [hunk 1]
```diff
@@ -64,3 +64,49 @@ exports.onCreateNode = ({ node }) => {
...
+            options: {},
+          },
+        ],
+      },
+    },
+    `gatsby-plugin-netlify-cms`,
+  ],
+}
+```
+This should fix the error above. Please notice that part of the options of this plugin `gatsby-remark-relative-images` is the name option "`assets`", that is the same name declared by the `gatsby-source-filesystem` above.
```
Comment (danielmahon) [inline README.md]:
```suggestion
```
Remove this line too 😉

**Comment by kinduff** [comment]
@danielmahon ready. Sorry for multiple commits, feel free to squash.

**Pull request merged**